foundation-ui/utils/ReactUtil.js

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
 
+function getUnwrappedElements(elements) {
+  if (Array.isArray(elements) && elements.length === 1
```

Comment:
Please document this case so that it's cleat to everyone that the util simply renders single item arrays.